### DISCUSSION TITLE
Getting issue on opening a new MAUI window

### DISCUSSION THREAD

**Discussion by slnsvr** [dotnet/maui#4381] [Q&A]
In MAUI sample project, getting unhandled exception at run time when trying to execute the below code 
Application.Current.OpenWindow(new Window { Page = new MainPage() }); 

In the eventvwr I can see the below exception.. 

Faulting application name: MauiSampleAppPreview4_3.exe, version: 1.0.0.0, time stamp: 0x61954b73 Faulting module name: KERNELBASE.dll, version: 10.0.19041.1466, time stamp: 0xe01c7650 – 

I have upgraded my visual studio to preview4 as well. Kindly suggest on the issue..

**Comment by jsuarezruiz** [comment]
Could you create an issue attaching a small sample where reproduce it?. Thanks!.
